backend/homework/9/src/main/java/com/example/springboot/service/VehicleService.java

```diff
@@ -0,0 +1,53 @@
+package com.example.springboot.service;
+
+import com.example.springboot.Logging.Logg;
```

Comment:
`Logging.Log` Instead of `Logging.Logg`
Make changes accordingly for better names